src/components/tabs/_tabs.scss

```diff
@@ -316,8 +314,11 @@
       width: auto;
     }
 
-    @include carbon--breakpoint(xlg) {
-      margin-left: 0;
+    @include carbon--breakpoint-down(md) {
```

Comment:
Hi @aledavila we no longer use `carbon--breakpoint-down()` in this file, to avoid issues _on_ the breakpoint. Please see #2128. Thanks!